hat/assets/js/apps/Iaso/utils/usersUtils.ts

```diff
@@ -62,7 +63,7 @@ export const getDisplayName = (
     // Some endpoint have user_name and some username (without the _, fun)
     const userName = user.user_name ?? user?.username;
```

Comment:
shouldn't it be `user?.user_name` to avoid `undefined.user-name` ?

Reply:
`user` cannot be null